### DISCUSSION TITLE
My-CS-Notebook/Jobs/2024spring/2024%E5%B9%B4%E6%9A%91%E6%9C%9F%E5%AE%9E%E4%B9%A0%E5%BA%94%E8%81%98%E8%AE%B0%E5%BD%95/

### DISCUSSION THREAD

**Discussion by giscus** [Zhang-Each/My-CS-Notebook#9] [General]
# My-CS-Notebook/Jobs/2024spring/2024%E5%B9%B4%E6%9A%91%E6%9C%9F%E5%AE%9E%E4%B9%A0%E5%BA%94%E8%81%98%E8%AE%B0%E5%BD%95/

https://zhang-each.github.io/My-CS-Notebook/Jobs/2024spring/2024%E5%B9%B4%E6%9A%91%E6%9C%9F%E5%AE%9E%E4%B9%A0%E5%BA%94%E8%81%98%E8%AE%B0%E5%BD%95/

**Comment by SophieCO2** [comment]
受教惹！

**Comment by csking2001** [comment]
师兄太强了，膜一个

**Comment by dellixx** [comment]
大佬 tql👍

**Comment by 123456789bhf** [comment]
看完佬的bg，终于能够心平气和的接受自己挂掉了
